Tasks/Maven/maven.ps1

```diff
@@ -126,6 +126,24 @@ $CCReportTask = "jacoco:report"
 
 Write-Verbose "SummaryFileCobertura = $summaryFileCobertura"
 
+if(Test-Path $reportDirectory)
+{
+    # delete any previous code coverage data 
+    rm -r $reportDirectory -force | Out-Null
```

Comment:
check if this fails correctly ie shouldnt terminate the script on failure... otherwise you should use the -erroraction option with the cmdlet